src/Sarif/SarifExtensionMethods.cs

```diff
@@ -15,8 +15,18 @@
 
 namespace Microsoft.CodeAnalysis.Sarif
 {
-    public static class ExtensionMethods
+    public static class SarifExtensions
```

Comment:
>SarifExtensions [](http://example.com/codeflow?start=24&length=15)

For publicly exposed extension methods classes, it's good to qualify the type name, to avoid collisions with this general name 'ExtensionMethods'.